java/pom.xml

```diff
@@ -34,6 +34,7 @@
     <connection>scm:git:https://github.com/googlei18n/libphonenumber.git</connection>
     <developerConnection>scm:git:git@github.com:googlei18n/libphonenumber.git</developerConnection>
     <url>https://github.com/googlei18n/libphonenumber/</url>
+    <tag>HEAD</tag>
```

Comment:
Yep, looks like it.

In the [first commit](https://github.com/googlei18n/libphonenumber/pull/1506/commits/717c7d7476669686ff2190da4c84938bc38bc0dc) it was <tag>v8.0.1</tag>, then the [second commit](https://github.com/googlei18n/libphonenumber/pull/1506/commits/806bbe817ee86496b8a740e674f82c26ecaf10c5) made it HEAD. This looks expected, and the file ended up at https://github.com/googlei18n/libphonenumber/releases/tag/v8.0.1 witht he v8.0.1 tag on this line.

==> No blockers for merging @kkeshava :)